tests/v2/actions/rbac/rbac.go

```diff
@@ -5,10 +5,9 @@ import (
 	"strings"
 
 	"github.com/rancher/norman/types"
+	rbacv2 "github.com/rancher/rancher/tests/v2/actions/kubeapi/rbac"
 	"github.com/rancher/shepherd/clients/rancher"
 	management "github.com/rancher/shepherd/clients/rancher/generated/management/v3"
-	rbacv2 "github.com/rancher/shepherd/extensions/kubeapi/rbac"
```

Comment:
for those curious, this is one file that had a line count difference compared to main. looks fine but not sure why the files are so different overall